src/posts/2020-01-27-stencil-getting-started.md

```diff
@@ -0,0 +1,70 @@
+---
+title: 'Aan de slag met Stencil'
+tags: Stencil, Web Components, Design System
+comments: true
+author: danny
+min_read: 4
+date: 2020-01-27
+---
+
+### Wat is Stencil?
+[Stencil](https://stenciljs.com/) is een compiler die Web Components (Custom Elements) genereert. Stencil combineert de beste concepten van de meest populaire frameworks in een simpele tool.
```

Comment:
Do we want to add `v1` to the Custom Elements?
e.g. `... Web Components (Custom Elements v1)`

Reply:
I think it is a good addition, even though v0 is deprecated this makes it immediate more obvious that v1 is used. Changed this in bf67141.